src/snowflake/cli/api/utils/definition_rendering.py

```diff
@@ -232,6 +239,26 @@ def _validate_env_section(env_section: dict):
             )
 
 
+def _get_referenced_vars_in_definition(
+    template_env: TemplatedEnvironment, definition: Definition
+):
+    referenced_vars = set()
+
+    def find_any_template_vars(element):
+        referenced_vars.update(template_env.get_referenced_vars(element))
+
+    traverse(definition, visit_action=find_any_template_vars)
+
+    return referenced_vars
+
+
+def _template_version_warning():
+    cc.warning(
```

Comment:
Don't add line breaks, it becomes brittle when we're using indentation. Same line will do. Soon I hope to make this API a bit better.

Reply:
Replaced them with spaces.